WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/AddressFormViewModelProtocol.swift

```diff
@@ -515,7 +523,8 @@ private extension AddressFormViewModel {
                 originalAddress != secondaryOriginalAddress &&
                 !showDifferentAddressForm
 
-                return .done(enabled: addressesAreDifferentButSecondAddressSwitchIsDisabled ||
+                return .done(enabled: isDoneButtonAlwaysEnabled ||
```

Comment:
Super nit: 
WDYT about using `guard` to return early when `isDoneButtonAlwaysEnabled` is true. Just like the existing guard with `!fields.useAsToggle` check.

This way, we can keep the field validation code separate.

Reply:
Good point, updated in https://github.com/woocommerce/woocommerce-ios/pull/8803/commits/3a107b171b36d5394c4796e73bfa4c55a588b62d